core/src/test/resources/HistoryServerExpectations/one_app_multi_attempt_json_expectation.json

```diff
@@ -11,6 +14,9 @@
     "completed" : true
   }, {
     "attemptId" : "1",
+    "startTimeEpoch" : 1426533910242,
+    "endTimeEpoch" : 1426533945177,
+    "lastUpdatedEpoch" : "",
```

Comment:
not seen that. It should be a 0, shoudn' it. Or send it over as a number in a string, where "" means not set? Client gets to parse